siliconcompiler/project.py

```
@@ -1379,7 +1379,7 @@ def snapshot(self, path: Optional[str] = None, jobname: Optional[str] = None,
 
     def show(self, filename: Optional[str] = None, screenshot: bool = False,
              extension: Optional[str] = None, tool: Optional[str] = None,
-             open: bool = False) -> str:
+             open: bool = False) -> Optional[str]:
```

Ruff (0.16.0)

[error] Function argument `open` is shadowing a Python builtin

(A002)